### DISCUSSION TITLE
Upgrade from 3.62 to 3.63

### DISCUSSION THREAD

**Discussion by SmsS4** [seaweedfs/seaweedfs#5348] [Q&A] [answered]
Hello
Should I do anything for upgrading from 3.62 to 3.63 because of #5215  ?
For example should I run weed fix before starting volumes?
Thanks.

**Comment by kmlebedev** [comment]
It is recommended that after stopping the volume and before updating, perform a `weed fix` of all volume files sequentially on each server

**Reply by SmsS4** [reply]
but weed fix sorts by offset in 3.62. so we should perform `weed fix` after update if I'm not mistaken

**Reply by kmlebedev** [reply, marked as answer]
yes
